bench/utils.py

```diff
@@ -301,7 +301,9 @@ def setup_app(app):
 
 
 def exec_cmd(cmd, cwd='.'):
-	import shlex
+	"""Executes 'cmd' as subprocess, then returns the return code.
+	If subprocess is successful, the return code is 0."""
+  import shlex
```

Comment:
```suggestion
	"""Executes cmd with cwd via subprocess.call and returns the process exit code"""
	import shlex
```

Give more info with lesser words. Fix indentation